doc/sphinx/source/libCEEDdev.md

```diff
@@ -34,8 +34,8 @@ for example.  All issues reported by `make tidy` should be fixed.
 
 Header inclusion for source files should follow the principal of 'include what you use' rather than relying upon transitive `#include` to define all symbols.
 
-Every symbol that is used in the source file `foo.c` should be defined in `foo.c`, `foo.h`, or in a header file ``` #include``d in one of these two locations.
-Please check your code by running the tool ``include-what-you-use ``` to see recommendations for changes to your source.
+Every symbol that is used in the source file `foo.c` should be defined in `foo.c`, `foo.h`, or in a header file `#include`d in one of these two locations.
+Please check your code by running the tool `include-what-you-use` to see recommendations for changes to your source.
```

Comment:
Should we be linking the repository and/or the vscode extension? https://marketplace.visualstudio.com/items?itemName=pokowaka.pokowaka-iwyu

Maybe we should add a make target for running it?

Reply:
Good call. I linked the repo and added make a target.